src/components/Posts/Posts.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Card } from '../Card/Card'
+
+
+type Post = {
+    postcardName: string,
+    postcardText: string,
+    img: string,
+}
+
+type Posts = {
+    posts : Post[]
+}
+export const Posts = ({posts}: Posts) => {
+    return (
+        
+        <div className="post-list__content">
+
+            {posts.map((post, id) => {
+
+                if(id === 0){
```

Comment:
Тут лучше именовать как есть - через index. Потому что id вносит неразбериху 
```suggestion
            {posts.map((post, index) => {

                if(index === 0){
```